code/bmpman/bmpman.cpp

```diff
@@ -188,6 +194,13 @@ float bitmap_lookup::get_channel_alpha(float u, float v)
 	return i2fl(Bitmap_data[(y*Width + x)*Num_channels + 3]) / 255.0f;
 }
 
+void clear_bm_lookup_cache() {
+	for(auto iter = bm_lookup_cache.begin(); iter != bm_lookup_cache.end(); iter++) {
+		free(iter->second);
+	}
```

Comment:
Looks like travis wants you to use modern loops. 

```
for(auto iter = bm_lookup_cache.begin(); iter != bm_lookup_cache.end(); iter++) {

        ^  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           (auto & iter : bm_lookup_cache)
```

Reply:
Looks like this is fixed in the current commit?